tests/testTriggers.go

```diff
@@ -7960,6 +7960,163 @@ func TriggerMetroDRMigrationSchedule(contexts *[]*scheduler.Context, recordChan
 	})
 }
 
+// AggrVolDepReplResizeOps crashes vol driver
+func TriggerAggrVolDepReplResizeOps(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	defer endLongevityTest()
+	startLongevityTest(AggrVolDepReplResizeOps)
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: AggrVolDepReplResizeOps,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+
+	setMetrics(*event)
+
+	stepLog := "Repl factor update and Volume Update on aggregated volume"
+	Step(stepLog, func() {
+		log.InfoD(stepLog)
+		log.Infof("Starting test case here !!")
+		// Validate minimum set of Storage nodes needed to create aggregated Volumes
+		sNodes := node.GetStorageNodes()
+		if len(sNodes) <= 3 {
+			err := fmt.Errorf("Node needs minimum of 3 Nodes to create aggregated volumes")
+			log.FailOnError(err, "is io running on the volume?")
+		}
+
+		aggrApplication := []string{"aggr-mysql", "postgres"}
```

Comment:
since it is part of longevity do not add apps in between, report failure if no aggregated volume is there,
Also can you create an fio spec with aggregated volumes

Reply:
sure , will do that